### ISSUE TITLE
AM Crashes on trying to change fixed system permissions

### ISSUE THREAD

**Issue by Atrate** [MuntashirAkon/AppManager#645]
- [x] I know what my device, OS and App Manager versions are
- [x] I know how to take logs
- [x] I know how to reproduce the issue which may not be specific to my device (not sure if it's not specific, but could be a thing for all fixed permissions)

**Describe the bug**
AM crashes trying to revoke a fixed system permission for a system app (root mode).

**To Reproduce**
Steps to reproduce the behaviour:
1. Open AM
2. Open the page for a system app, e.g. `com.onyx.appmarket`
3. Try to revoke a permission, such as `android.permission.CAMERA`
4. AM Crashes

**Expected behavior**
The permission should be denied and AM should not crash.

**Crash logs**
```
java.lang.SecurityException: Non-System UID cannot revoke system fixed permission android.permission.CAMERA for package com.onyx.appmarket
    at android.os.Parcel.createException(Parcel.java:2071)
    at android.os.Parcel.readException(Parcel.java:2039)
    at android.os.Parcel.readException(Parcel.java:1987)
    at android.content.pm.IPackageManager$Stub$Proxy.revokeRuntimePermission(IPackageManager.java:5504)
    at io.github.muntashirakon.AppManager.servermanager.PackageManagerCompat.revokePermission(PackageManagerCompat.java:124)
    at io.github.muntashirakon.AppManager.details.AppDetailsViewModel.setPermission(AppDetailsViewModel.java:393)
    at io.github.muntashirakon.AppManager.details.AppDetailsFragment$AppDetailsRecyclerAdapter.lambda$null$22$AppDetailsFragment$AppDetailsRecyclerAdapter(AppDetailsFragment.java:1353)
    at io.github.muntashirakon.AppManager.details.-$$Lambda$AppDetailsFragment$AppDetailsRecyclerAdapter$ebSZkn7bHyfqKqD18v0XtWPMWr4.run(Unknown Source:10)
    at java.lang.Thread.run(Thread.java:919)
 Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.pm.permission.PermissionManagerService.revokeRuntimePermission(PermissionManagerService.java:2345)
	at com.android.server.pm.permission.PermissionManagerService.access$1400(PermissionManagerService.java:127)
	at com.android.server.pm.permission.PermissionManagerService$PermissionManagerServiceInternalImpl.revokeRuntimePermission(PermissionManagerService.java:3127)
	at com.android.server.pm.PackageManagerService.revokeRuntimePermission(PackageManagerService.java:5778)
	at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:2321)


Device Info:
App version: 2.5.24-PRE
App version code: 383
Android build version: 1834
Android release version: 10
Android SDK version: 29
Android build ID: 2021-03-18_09-20_3.1_ad716f8
Device brand: ONYX
Device manufacturer: ONYX
Device name: Note3
Device model: Note3
Device product name: Note3
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-US
In-App Language: auto
Mode: auto
```

**Device info**
 - Device: Onyx Boox Note 3
 - OS Version: Android 10
 - App Manager Version: v2.5.24-PRE
 - Mode: root

**Comment by MuntashirAkon** [comment]
Please do the same but with the latest version i.e. v2.6.0 or later.

**Comment by Atrate** [comment]
The same happens on AM 2.6.5.1

**Comment by MuntashirAkon** [comment]
Fixed in 0b571a38
